docs/srt-live-transmit.md

```diff
@@ -287,4 +287,64 @@ When leaving the LAN for testing, please keep an eye on statistics and make sure
 
 If you perform tests on the public Internet, consider checking your firewall rules. The **SRT listener** must be reachable on the chosen UDP port. Same applies to routers using NAT. Please set a port forwarding rule with protocol UDP to the local IP address of the **SRT listener**.
 
-The direction of initiating the stream doesn't need to be the same as the stream direction. The sender of a stream can be a **SRT listener** or **SRT caller** as long as the receiving end uses the opposite mode. Typically you use the **SRT listener** on the end, which is easier to configure in terms of firewall/router setup. It also makes sense to leave the Sender in listener mode when trying to connect from various end points with possibly unknown IP addresses.
\ No newline at end of file
+The direction of initiating the stream doesn't need to be the same as the stream direction. The sender of a stream can be a **SRT listener** or **SRT caller** as long as the receiving end uses the opposite mode. Typically you use the **SRT listener** on the end, which is easier to configure in terms of firewall/router setup. It also makes sense to leave the Sender in listener mode when trying to connect from various end points with possibly unknown IP addresses.
+
+## UDP Performance
+
+Performance issues concerning reading from UDP medium were reported in #933 and #762.
+
+The ddicated research showed that at high and bursty data rates (~60 Mbps)
+the `epoll_wait(udp_socket)` is not fast enough to signal about the possibility
+to read from a socket. It results in loosing data when the input bitrate is very high (above 20 Mbps).
+
+Waiting on a UDP socket with `::select(...)` works perfect,
+but it can't be used in the current implementation of the `srt-live-transmit`
+due to its design.
```

Comment:
You are touching here a very spanning topic and therefore better not mention it, or simply get straight to the point.

The point isn't that the "architecture" doesn't let you use `::select` but that the architecture based on the non-blocking mode mandates the use of a common selection function - and `::select` cannot be used to poll on a UDP and an SRT socket simultaneously. Although it would be possible to "translate" the readiness signal from SRT epoll to `::select` or other native pollers (epoll/kqueue/WaitForMultipleObjects), it is complicated as for a simple application and requires an extra thread.